client/extensions/woocommerce/state/sites/setup-choices/selectors.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import { get, isObject } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { LOADING } from 'woocommerce/state/constants';
+
+const getRawSetupChoices = ( state, siteId ) => {
```

Comment:
Agreed! I was copying what we did in settings, but this is simple enough. Fixed in https://github.com/Automattic/wp-calypso/pull/14339/commits/fc929a9089c59738059bec4d882378d93712696e